inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/JobConstants.java

```diff
@@ -29,6 +29,7 @@
     public static final String JOB_RETRY = "job.retry";
 
     public static final String JOB_SOURCE = "job.source";
+    public static final String JOB_SOURCE_TYPE = "job.sourceType";
```

Comment:
what's difference between `job.source` and `job.sourceType`?

Reply:
source means the class name for source, I will change the param name for better understanding